containers/docker/pwpush/Dockerfile

```diff
@@ -0,0 +1,66 @@
+# pwpush-ephemeral
+FROM ruby:3.2-alpine AS build-env
+
+LABEL maintainer='pglombardo@hey.com'
+
+# Required packages
+RUN apk upgrade --no-cache \ 
+    && apk add --no-cache build-base git curl tzdata zlib-dev nodejs yarn libc6-compat sqlite-dev mariadb-dev libpq-dev
+
+ENV APP_ROOT=/opt/PasswordPusher PATH=${APP_ROOT}:${PATH} HOME=${APP_ROOT}
+
+RUN mkdir -p ${APP_ROOT}
+COPY ./ ${APP_ROOT}/
+
+WORKDIR ${APP_ROOT}
```

Comment:
`WORKDIR` implies `mkdir`.  So you can do as following instead:

```dockerfile
WORKDIR ${APP_ROOT}
COPY . .     # maybe need a . ./
```

Reply:
Fine as well 

For Doc purpose: [https://docs.docker.com/engine/reference/builder/#workdir](https://docs.docker.com/engine/reference/builder/#workdir)